Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/recovery/core/RecoveryCertificateRepositoryTest.kt

```diff
@@ -116,7 +116,7 @@ class RecoveryCertificateRepositoryTest : BaseTest() {
             recoveryCertificate.qrCodeToDisplay.options.correctionLevel shouldBe ErrorCorrectionLevel.M
         }
 
-        testStorage.first().recoveryCertificateQrCode shouldBe RecoveryQrCodeTestData.recoveryQrCode1
+        // testStorage.first().recoveryCertificateQrCode shouldBe RecoveryQrCodeTestData.recoveryQrCode1
```

Comment:
Leftover ? or should be adapted?

Reply:
Adapted test to verify `storage.save(any())` was executed